test/drenv/addons/rook/external_cluster/start.py

```
@@ -0,0 +1,200 @@
+    )
+
+
+def _get_rook_ceph_external_cluster_details(podname, cluster):
+    try:
+        kubectl.exec(
+            podname,
+            f"--namespace={ROOK_CEPH_NAMESPACE}",
+            "--",
+            "curl",
+            "-o",
+            "/tmp/create-external-cluster-resources.py",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/create-external-cluster-resources.py"

(S108)